### PR TITLE
preview server: add file watch

### LINKED ISSUE #535: Staging server: add watch build task
So users can edit content files and have the gulp task re-run.

### PR DESCRIPTION
**Pull request by ebidel**
Fixes #535.

### CODE REVIEW

File: content/index.html [hunk 1]
```diff
@@ -96,42 +96,42 @@ <h3 class="join-us__header-title">
...
-                ,*,.                                        .,**////*          
-                ,*.                                         .,*.....           
-               .**.                                         .,*.               
-               .,************,,,,,..........                .*,.               
-                         ,*////**,,,***********,,,,,,.......,*,                
-                         ,*////*.           .*/////*,,,*******,                
-                         ,*////*.            ,*////,.                          
-                         .*////*.            ,*///*,                           
+                   ,*,,,,,,,***********,,,,,.....
+                    ,*,.        ..........,,,,,,****,.
```
Comment (ebidel) [inline content/index.html]: oh hey webby.

**Comment by robdodson** [comment]
What do you think about having it always run watch? Is there a scenario when I wouldn't want that?

**Comment by ebidel** [comment]
I'm not opposed, but personally I hate when command likes this spawn a bunch of different watchers, live reloads, and servers.

**Comment by robdodson** [comment]
@ebidel normally I agree but in this case, because we're not transpiling JS or compiling SCSS, it seems like the primary use case of the staging server is to edit/refresh a doc so it seems like we may want it.

File: README.md [hunk 2]
```diff
@@ -7,31 +7,42 @@ web.dev is the ultimate resource for developers of all backgrounds to learn, cre
...
+npm i
+# yarn
+```
 
-## Previewing the site
+### Previewing the site
 
-To create/edit content and preview a page locally, start the "preview server":
+To create/edit content and preview a page locally as you make edits, start the "preview server":
+author.
```
Comment (robdodson) [inline README.md]:
```suggestion

```

File: package.json [hunk 3]
```diff
@@ -10,9 +10,10 @@
     "node": ">=10"
   },
   "scripts": {
-    "build": "./lib/index.js",
+    "clean": "gulp clean",
+    "build": "gulp build",
+    "watch": "gulp watch & yarn start",
```
Comment (robdodson) [inline package.json]:
```suggestion
     "dev": "gulp watch & npm start",
```

File: README.md [hunk 4]
```diff
@@ -7,31 +7,42 @@ web.dev is the ultimate resource for developers of all backgrounds to learn, cre
...
 
 ```shell
-npm run start
+npm run watch
```
Comment (robdodson) [inline README.md]:
```suggestion
 npm run dev
```

File: README.md [hunk 5]
```diff
@@ -7,31 +7,42 @@ web.dev is the ultimate resource for developers of all backgrounds to learn, cre
...
+# yarn watch
```
Comment (robdodson) [inline README.md]:
```suggestion
 # yarn dev
```

File: package.json (hunk 3 above)
Comment (robdodson) [reply package.json]: Suggest changing this to "dev" to match what other CLI's like Next.js use for similar functionality.

Comment (robdodson) [reply package.json]: I don't think this exits cleanly. When you ctrl-c it kills the server, but gulp watch continues to run in the background.

**Comment by robdodson** [comment]
I think you may want to use [concurrently](https://www.npmjs.com/package/concurrently) to run both watch and start at the same time.

File: package.json (hunk 3 above)
Comment (ebidel) [reply package.json]: done

**Comment by ebidel** [comment]
PTAL

**Comment by robdodson** [comment]
LGTM!

**Pull request merged**